### PR TITLE
mold 1.3.0

### PR DESCRIPTION
**Pull request by sabify**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by equal-l2** [comment]
The author wants binary packages to use their bundled TBB, which contains a patch:

```
Note for those who create mold binary packages: if you are building mold for binary distribution, please link the bundled libtbb statically (which is default) or rebuild your distro's libtbb package with https://github.com/oneapi-src/oneTBB/pull/824 so that mold's Link-Time Optimization (LTO) works reliably under a heavy load.
```
https://github.com/rui314/mold/releases/tag/v1.3.0

(I wonder if Homebrew's policy allow it, though)

**Comment by carlocab** [comment]
We don't want to link `tbb` statically, but we can look at applying the patch to `tbb`.

**Comment by sabify** [comment]
> We don't want to link `tbb` statically, but we can look at applying the patch to `tbb`.

@carlocab Could you please put that `tbb` patch into action too?

File: Formula/mold.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Mold < Formula
   uses_from_macos "zlib"
 
   on_macos do
-    depends_on "llvm" => [:build, :test] if DevelopmentTools.clang_build_version <= 1200
+    depends_on "llvm" if DevelopmentTools.clang_build_version <= 1200
```
Comment (carlocab) [inline Formula/mold.rb:22]:
```suggestion
    depends_on "llvm" => [:build, :test] if DevelopmentTools.clang_build_version <= 1200
```

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**